LEGO1/lego/legoomni/include/legogamestate.h

```diff
@@ -172,13 +172,11 @@ class LegoGameState {
 
 	Act GetCurrentAct() { return m_currentAct; }
 	Act GetLoadedAct() { return m_loadedAct; }
-	Area GetCurrentArea() { return m_currentArea; }
 	Area GetPreviousArea() { return m_previousArea; }
 	Area GetUnknown0x42c() { return m_unk0x42c; }
 	History* GetHistory() { return &m_history; }
 
 	void SetDirty(MxBool p_isDirty) { m_isDirty = p_isDirty; }
-	void SetCurrentArea(Area p_currentArea) { m_currentArea = p_currentArea; }
```

Comment:
See the comment in line 211 (old file). I verified a number of BETA10 matches and none of them used a getter or setter.